src/node_modules/app/machines/builder/formconfig.options.js

```diff
@@ -0,0 +1,41 @@
+// eslint-disable-next-line node/no-unpublished-import
+import { assign } from 'xstate';
+
+function setTab (ctx, event) {
+	ctx.selectedRequestTab.set(event.selectedRequestTab);
+	console.log(event);
+	return ctx
+}
+
+export const configurationOptions = {
+	actions: {
+		/**
+		 * Toggles auto-executing requests when the query is valid
+		 */
+		toggleAutoExecute: ctx => ctx.runQueryOnSelect.update(v => !v),
```

Comment:
I'd add 

```js
const toggle = v => !v;
```

on top and I'd reuse like 

```suggestion
		toggleAutoExecute: ctx => ctx.runQueryOnSelect.update(toggle),
```

in this and the other actions, to avoid having 6 identical functions in memory.

(I'm adding toggle to `@svizzle/utils` so this could be imported from there later on)